src/dialogxml/dialogs/strchoice.cpp

```diff
@@ -12,10 +12,15 @@
 #include <algorithm>
 
 #include <boost/lexical_cast.hpp>
+#include <boost/algorithm/string/case_conv.hpp>
 
 #include "dialogxml/widgets/field.hpp"
 #include "fileio/resmgr/res_dialog.hpp"
 #include "sounds.hpp"
+#include "gfx/render_shapes.hpp"
+
+const sf::Color HILITE_COLOUR = Colours::LIGHT_GREEN;
+const sf::Color NORMAL_COLOUR = Colours::WHITE;
```

Comment:
This is incorrect. The normal colour is not white. It is white in the game and PC editor and black in the scenario editor. I'm pretty sure I have a `defClr` or similar somewhere that handles this.

Reply:
```
	if(!attrs.count("color") && !attrs.count("colour") && getDialog()->getBg() == cDialog::BG_DARK)
		setColour(sf::Color::White);
```
Is it this, in message.cpp?

Reply:
oh `parent.getDefTextClr()`